src/components/codeBlock.tsx

```diff
@@ -73,6 +103,59 @@ type KeywordSelectorProps = {
   keyword: string;
 };
 
+function OrgAuthTokenCreator() {
+  const codeContext = useContext(CodeContext);
+
+  const [tokenState, setTokenState] = useState<'none' | 'loading' | 'success' | 'error'>(
+    'none'
+  );
+  const [token, setToken] = useState(null);
+  const [sharedSelection] = codeContext.sharedKeywordSelection;
+
+  const {codeKeywords} = useContext(CodeContext);
+
+  const choices = codeKeywords?.PROJECT;
+
+  // When not signed in, we just show a placeholder, as the user can't generate a token in this case
+  if (!codeKeywords.USER) {
+    return <Fragment>sntrys_YOUR_TOKEN_HERE</Fragment>;
+  }
+
+  const currentSelectionIdx = sharedSelection.PROJECT ?? 0;
+  const currentSelection = choices[currentSelectionIdx];
```

Comment:
Does it prompt you for which organization to create the token for?

I've been thinking about how we can generally make this experience better, so maybe the context would be a lot more obvious

Reply:
this just takes the org that is selected - in the UI we moved the `__ORG___` above this selector to accommodate this - it would take `example-org` in this case:

<img width="767" alt="Screenshot 2023-07-10 at 15 36 54" src="https://github.com/getsentry/sentry-docs/assets/2411343/de5f3b5e-3faa-4d5b-9322-e57a42543c3f">